src/test/txvalidation_tests.cpp

```diff
@@ -53,9 +53,8 @@ BOOST_FIXTURE_TEST_CASE(tx_mempool_reject_coinbase, TestChain100Setup)
     BOOST_CHECK(state.IsInvalid());
     BOOST_CHECK_EQUAL(state.GetRejectReason(), "coinbase");
 
-    int nDoS;
-    BOOST_CHECK_EQUAL(state.IsInvalid(nDoS), true);
-    BOOST_CHECK_EQUAL(nDoS, 100);
+    BOOST_CHECK_EQUAL(state.IsInvalid(), true);
+    BOOST_CHECK(state.GetReason() == ValidationInvalidReason::CONSENSUS);
```

Comment:
Also BOOST_CHECK_EQUAL?

Reply:
Doesn't compile as GetReason() can't be printed.

Reply:
Not worthy of revisiting here, but fyi I added a facility here to support `BOOST_CHECK_EQUAL` for enum: https://github.com/bitcoin/bitcoin/pull/13076/files#diff-4a0cb5ad5e93d187a1b065a99bbae143